lib/conduit_mqtt.ex

```diff
@@ -52,10 +50,10 @@ defmodule ConduitMQTT do
 
   def publish(broker, message, _config, opts) do
     #props = ConduitMQTT.Props.get(message) #TODO MQTT opts
+
     with_client_id(broker, fn client_id ->
-      qos = Message.get_header(message, "qos")
-      Logger.info("Publishing from client_id #{client_id} to #{message.destination} with qos #{qos}")
-      Tortoise.publish_sync(client_id, message.destination, message.body, [qos: qos]) #TODO: how to do qos
+      Logger.debug("Publishing from broker #{broker} client_id #{client_id} to #{message.destination} with opts #{inspect(opts)}")
```

Comment:
This logger should go away. The intended means to log stuff like this is with plugs.